tasks/start/start_action.js

```diff
@@ -6,7 +6,9 @@ module.exports = function (plugin, run, options) {
 
   const cmd = 'node';
   const script = join('scripts', 'kibana.js');
-  let args = [script, '--dev', '--plugin-path', plugin.root];
+  const nodeOptions = process.env.NODE_OPTIONS;
+
+  let args = [nodeOptions, script, '--dev', '--plugin-path', plugin.root].filter(Boolean);
```

Comment:
The args array escapes each item so this prevents you from passing multiple options. Perhaps we should use [`argv-split`](https://www.npmjs.com/package/argv-split) to get the individual flags and then merge that array in.

```sh
NODE_OPTIONS="--debug-brk --inspect" npm start
node: bad option: --debug-brk --inspect ## because --debug-brk and --inspect are considered as one option 
```

Reply:
wouldn't `...nodeOptions.split(' ')` do the trick?

Reply:
Not for `NODE_OPTIONS='--eval="console.log(\"I am a script\")"` or some other option value that has double quotes.

Reply:
https://www.npmjs.com/package/argv-split#why